cloudpickle/cloudpickle.py

```diff
@@ -249,6 +252,93 @@ def _walk_global_ops(code):
                 yield op, instr.arg
 
 
+def _memoryview_from_bytes(data, format, readonly):
+    if not readonly:
```

Comment:
I have written something very similar for pandas: https://github.com/pandas-dev/pandas/blob/master/pandas/util/move.c#L160

The test suite exercises some interesting edge cases we may want to replicate: https://github.com/pandas-dev/pandas/blob/master/pandas/tests/util/test_util.py#L324

Reply:
Thanks I will have a look.